__tests__/api.test.js

```diff
@@ -243,3 +243,35 @@ describe("PATCH /api/articles/:article_id", () => {
       });
   });
 });
+
+describe("DELETE /api/comments/comment_id", () => {
+  test("204: should delete the comment with the associated comment_id and return an empty body", () => {
+    return request(app)
+      .delete("/api/comments/1")
+      .expect(204)
+      .then((response) => {
+        const { body } = response;
+        expect(body).toEqual({});
```

Comment:
expecting a 204 is enough here and we don't need to create and send back an empty object.